Error when starting:    SyntaxError: invalid syntax

**Additional context**
It seems that [kombu has a bug on python 3.7](https://github.com/celery/kombu/issues/841), which would cause the following error. This bug fixed on kombu 4.2.0 but taiga still using kombu 4.0.2, so please consider downgrade the python to 3.6

**Describe the bug**
error when starting taiga-docker
```
taiga-back_1   | SyntaxError: invalid syntax
taiga-back_1   | [2019-03-25 01:47:32 +0000] [18] [ERROR] Exception in worker process
taiga-back_1   | Traceback (most recent call last):
taiga-back_1   |   File "/usr/local/lib/python3.7/site-packages/gunicorn/workers/base.py", line 135, in load_wsgi
taiga-back_1   |     self.wsgi = self.app.wsgi()
taiga-back_1   |   File "/usr/local/lib/python3.7/site-packages/gunicorn/app/base.py", line 67, in wsgi
taiga-back_1   |     self.callable = self.load()
taiga-back_1   |   File "/usr/local/lib/python3.7/site-packages/gunicorn/app/wsgiapp.py", line 65, in load
taiga-back_1   |     return self.load_wsgiapp()
taiga-back_1   |   File "/usr/local/lib/python3.7/site-packages/gunicorn/app/wsgiapp.py", line 52, in load_wsgiapp
taiga-back_1   |     return util.import_app(self.app_uri)
taiga-back_1   |   File "/usr/local/lib/python3.7/site-packages/gunicorn/util.py", line 352, in import_app
taiga-back_1   |     __import__(module)
taiga-back_1   |   File "/usr/src/taiga-back/taiga/__init__.py", line 19, in <module>
taiga-back_1   |     from . import celery
taiga-back_1   |   File "/usr/src/taiga-back/taiga/celery.py", line 21, in <module>
taiga-back_1   |     from celery import Celery
taiga-back_1   |   File "<frozen importlib._bootstrap>", line 1032, in _handle_fromlist
taiga-back_1   |   File "/usr/local/lib/python3.7/site-packages/celery/local.py", line 509, in __getattr__
taiga-back_1   |     module = __import__(self._object_origins[name], None, None, [name])
taiga-back_1   |   File "/usr/local/lib/python3.7/site-packages/celery/app/__init__.py", line 5, in <module>
taiga-back_1   |     from celery import _state
taiga-back_1   |   File "/usr/local/lib/python3.7/site-packages/celery/_state.py", line 15, in <module>
taiga-back_1   |     from celery.utils.threads import LocalStack
taiga-back_1   |   File "/usr/local/lib/python3.7/site-packages/celery/utils/__init__.py", line 10, in <module>
taiga-back_1   |     from .nodenames import worker_direct, nodename, nodesplit
taiga-back_1   |   File "/usr/local/lib/python3.7/site-packages/celery/utils/nodenames.py", line 7, in <module>
taiga-back_1   |     from kombu.entity import Exchange, Queue
taiga-back_1   |   File "/usr/local/lib/python3.7/site-packages/kombu/entity.py", line 6, in <module>
taiga-back_1   |     from .abstract import MaybeChannelBound, Object
taiga-back_1   |   File "/usr/local/lib/python3.7/site-packages/kombu/abstract.py", line 6, in <module>
taiga-back_1   |     from .connection import maybe_channel
taiga-back_1   |   File "/usr/local/lib/python3.7/site-packages/kombu/connection.py", line 893
taiga-back_1   |     return self.transport.implements.async
```

**To Reproduce**
docker-compose file:
```
version: '2'

services:
  taiga-back:
    image: monogramm/docker-taiga-back-base:latest
    depends_on:
      - postgres

    environment:
      TAIGA_HOSTNAME: localhost
      TAIGA_DB_HOST: postgres
      TAIGA_DB_NAME: taigadb
      TAIGA_DB_USER: taiga
      TAIGA_DB_PASSWORD: password

    volumes:
      - ./media:/usr/src/taiga-back/media
      - ./taiga-conf:/taiga

  taiga-front:
    image: monogramm/docker-taiga-front-base:latest
    ports:
      - 2380:80
    depends_on:
      - taiga-back
    environment:
      TAIGA_BACK_HOST: taiga-back
      TAIGA_BACK_PORT: 8001

  postgres:
    image: postgres
    environment:
      POSTGRES_DB: taigadb
      POSTGRES_USER: taiga
      POSTGRES_PASSWORD: password
    ports:
      - 5432
    volumes:
      - ./pgdata:/var/lib/postgresql/data
```

**Docker (please complete the following information):**
 - Docker version 18.09.1, build 4c52b90
 - Image: latest

--- a/.travis.yml
+++ b/.travis.yml
@@ -17,6 +17,12 @@ before_script:
 
 script:
   - docker build -t "$image" "$dir"
+  - docker-compose build
+  - docker-compose up -d
+  - docker-compose ps
+  - docker-compose logs
+  - docker-compose ps | grep "$image"
+  - docker-compose logs | grep ""
 
 after_script:
   - docker images

--- a/.travis.yml
+++ b/.travis.yml
@@ -21,8 +21,8 @@ script:
   - docker-compose up -d
   - docker-compose ps
   - docker-compose logs
-  - docker-compose ps | grep "$image"
-  - docker-compose logs | grep ""
+  - travis_retry docker-compose build && docker-compose ps | grep "$image"
+  - docker-compose logs
 
 after_script:
   - docker images

--- a/.travis.yml
+++ b/.travis.yml
@@ -14,6 +14,7 @@ before_script:
   - name="taiga-back"
   - image="monogramm/docker-taiga-back-base:travis-${VERSION}-${VARIANT}"
   - dir="images/${VERSION}/${VARIANT}/"
+  - container_name="taiga_back"
 
 script:
   - docker build -t "$image" "$dir"
@@ -21,7 +22,7 @@ script:
   - docker-compose up -d
   - docker-compose ps
   - docker-compose logs
-  - travis_retry docker-compose build && docker-compose ps | grep "$image"
+  - travis_retry docker-compose build && docker-compose ps | grep "$container_name"
   - docker-compose logs
 
 after_script:

--- a/.travis.yml
+++ b/.travis.yml
@@ -11,25 +11,17 @@ branches:
 
 before_script:
   - env | sort
-  - name="taiga-back"
-  - image="monogramm/docker-taiga-back-base:travis-${VERSION}-${VARIANT}"
-  - dir="images/${VERSION}/${VARIANT}/"
   - container_name="taiga_back"
 
 script:
-  - docker build -t "$image" "$dir"
   - docker-compose build
-  - docker-compose up -d
+  - travis_retry docker-compose up -d && docker-compose ps | grep "$container_name"
   - docker-compose ps
-  - docker-compose logs
-  - travis_retry docker-compose build && docker-compose ps | grep "$container_name"
   - docker-compose logs
 
 after_script:
   - docker images
-  - docker run --name "$name" -d "$image"
-  - docker ps
-  - docker logs "$name"
+  - docker ps -a
 
 notifications:
   email: false

--- a/README.md
+++ b/README.md
@@ -26,6 +26,17 @@ Taiga is a project management platform for startups and agile developers & desig
 > [taiga.io](https://taiga.io)
 
 
+## Supported tags
+
+https://hub.docker.com/r/monogramm/docker-nextcloud/
+
+-	`3.3`, `3.3-alpine` (*3.3/alpine/Dockerfile*)
+-	`3.4`, `3.4-alpine`, `3`, `3-alpine` (*3.4/alpine/Dockerfile*)
+-	`4.0`, `4.0-alpine` (*4.0/alpine/Dockerfile*)
+-	`4.1`, `4.1-alpine` (*4.1/alpine/Dockerfile*)
+-	`4.2-alpine`, `4.2`, `4-alpine`, `4`, `alpine`, `latest` (*4.2/alpine/Dockerfile*)
+
+
 ## Build Docker image
 
 To generate docker images from the template, execute `update.sh` script.
@@ -74,7 +85,7 @@ See [local.py.example](https://github.com/taigaio/taiga-back/blob/master/setting
 Your database name (REQUIRED)
 
 Examples:
-```
+```yml
 TAIGA_DB_NAME=taiga
 TAIGA_DB_NAME=taigadb
 ```
@@ -86,7 +97,7 @@ TAIGA_DB_NAME=taigadb
 Your database hostname (REQUIRED)
 
 Examples:
-```
+```yml
 TAIGA_DB_HOST=taigadb
 TAIGA_DB_HOST=taigadb.company.com
 ```
@@ -98,7 +109,7 @@ TAIGA_DB_HOST=taigadb.company.com
 Your database user (REQUIRED)
 
 Examples:
-```
+```yml
 TAIGA_DB_USER=taiga
 TAIGA_DB_USER=taigadb
 ```
@@ -110,7 +121,7 @@ TAIGA_DB_USER=taigadb
 Your database user passsword (REQUIRED)
 
 Examples:
-```
+```yml
 TAIGA_DB_PASSWORD=somethingsecure
 ```
 
@@ -121,7 +132,7 @@ TAIGA_DB_PASSWORD=somethingsecure
 Your service hostname (REQUIRED). Remember to set it in the front client too.
 
 Examples:
-```
+```yml
 TAIGA_HOSTNAME=localhost
 TAIGA_HOSTNAME=taiga.company.com
 ```
@@ -133,78 +144,34 @@ TAIGA_HOSTNAME=taiga.company.com
 Enable email server configuration
 
 Examples:
-```
+```yml
 TAIGA_ENABLE_EMAIL=False
+```
+```yml
 TAIGA_ENABLE_EMAIL=True
-```
-
-### TAIGA_EMAIL_FROM
-
-*Default value*:
-
-The default 'From' email
-
-Examples:
-```
-TAIGA_EMAIL_FROM=no-reply@company.com
-```
-
-### TAIGA_EMAIL_USE_TLS
-
-*Default value*: `False`
-
-Use STARTTLS to connect email server
-
-Examples:
-```
+TAIGA_EMAIL_FROM=no-reply@gmail.com
 TAIGA_EMAIL_USE_TLS=False
-TAIGA_EMAIL_USE_TLS=True
-```
-
-### TAIGA_EMAIL_HOST
-
-*Default value*:
-
-The email server hostname
-
-Examples:
-```
-TAIGA_EMAIL_HOST=mail.company.com
 TAIGA_EMAIL_HOST=smtp.gmail.com
-```
-
-### TAIGA_EMAIL_PORT
-
-*Default value*:
-
-The email server port
-
-Examples:
-```
-TAIGA_EMAIL_PORT=25
 TAIGA_EMAIL_PORT=465
+TAIGA_EMAIL_USER=user.taiga@gmail.com
+TAIGA_EMAIL_PASS=usertaigagmailappsecret
+```
+```yml
+TAIGA_ENABLE_EMAIL=True
+TAIGA_EMAIL_FROM=no-reply@company.com
+TAIGA_EMAIL_USE_TLS=True
+TAIGA_EMAIL_HOST=mail.company.com
 TAIGA_EMAIL_PORT=587
-```
-
-### TAIGA_EMAIL_USER
-
-*Default value*:
-
-The email server user
-
-Examples:
-```
 TAIGA_EMAIL_USER=taiga@company.com
+TAIGA_EMAIL_PASS=somethingsecure
 ```
-
-### TAIGA_EMAIL_PASS
-
-*Default value*:
-
-The email server user passsword
-
-Examples:
-```
+```yml
+TAIGA_ENABLE_EMAIL=True
+TAIGA_EMAIL_FROM=no-reply@company.com
+TAIGA_EMAIL_USE_TLS=False
+TAIGA_EMAIL_HOST=mail.company.com
+TAIGA_EMAIL_PORT=25
+TAIGA_EMAIL_USER=taiga@company.com
 TAIGA_EMAIL_PASS=somethingsecure
 ```
 
@@ -215,7 +182,7 @@ TAIGA_EMAIL_PASS=somethingsecure
 The default administrator password
 
 Examples:
-```
+```yml
 TAIGA_ADMIN_PASSWORD=somethingverysecure
 ```
 
@@ -226,7 +193,7 @@ TAIGA_ADMIN_PASSWORD=somethingverysecure
 Activate SSL. Remember to enable it in the front client too.
 
 Examples:
-```
+```yml
 TAIGA_SSL=False
 TAIGA_SSL=True
 ```
@@ -238,7 +205,7 @@ TAIGA_SSL=True
 Activate SSL through a reverse proxy. Remember to enable it in the front client too.
 
 Examples:
-```
+```yml
 TAIGA_SSL_BY_REVERSE_PROXY=False
 TAIGA_SSL_BY_REVERSE_PROXY=True
 ```
@@ -250,7 +217,7 @@ TAIGA_SSL_BY_REVERSE_PROXY=True
 Secret key used for encryption.
 
 Examples:
-```
+```yml
 TAIGA_SECRET_KEY=somethingreallysecureandrandom
 ```
 
@@ -261,9 +228,9 @@ TAIGA_SECRET_KEY=somethingreallysecureandrandom
 Enable Taiga debug mode.
 
 Examples:
-```
-TAIGA_DEBUG=false
-TAIGA_DEBUG=true
+```yml
+TAIGA_DEBUG=False
+TAIGA_DEBUG=True
 ```
 
 ### TAIGA_TEMPLATE_DEBUG
@@ -273,10 +240,9 @@ TAIGA_DEBUG=true
 Enable Taiga template debug mode.
 
 Examples:
-```
-TAIGA_TEMPLATE_DEBUG=false
-TAIGA_TEMPLATE_DEBUG=true
-TAIGA_TEMPLATE_DEBUG=
+```yml
+TAIGA_TEMPLATE_DEBUG=False
+TAIGA_TEMPLATE_DEBUG=True
 ```
 
 ### TAIGA_PUBLIC_REGISTER_ENABLED
@@ -286,7 +252,7 @@ TAIGA_TEMPLATE_DEBUG=
 Enable Taiga registration.
 
 Examples:
-```
+```yml
 TAIGA_PUBLIC_REGISTER_ENABLED=False
 TAIGA_PUBLIC_REGISTER_ENABLED=True
 ```
@@ -298,7 +264,7 @@ TAIGA_PUBLIC_REGISTER_ENABLED=True
 Enable Taiga sitemap.
 
 Examples:
-```
+```yml
 TAIGA_SITEMAP_ENABLED=False
 TAIGA_SITEMAP_ENABLED=True
 ```
@@ -310,10 +276,10 @@ TAIGA_SITEMAP_ENABLED=True
 Enable Taiga feedback. Remember to enable it in the front client too.
 
 Examples:
-```
+```yml
 TAIGA_FEEDBACK_ENABLED=False
 ```
-```
+```yml
 TAIGA_FEEDBACK_ENABLED=True
 TAIGA_FEEDBACK_EMAIL=support@taiga.io
 ```
@@ -321,7 +287,7 @@ TAIGA_FEEDBACK_EMAIL=support@taiga.io
 TAIGA_FEEDBACK_ENABLED=True
 TAIGA_FEEDBACK_EMAIL=taiga@company.com
 ```
-```
+```yml
 TAIGA_FEEDBACK_ENABLED=True
 TAIGA_FEEDBACK_EMAIL=contact@company.com
 ```
@@ -333,7 +299,7 @@ TAIGA_FEEDBACK_EMAIL=contact@company.com
 Enable Taiga statistics.
 
 Examples:
-```
+```yml
 TAIGA_STATS_ENABLED=False
 TAIGA_STATS_ENABLED=True
 ```
@@ -345,10 +311,10 @@ TAIGA_STATS_ENABLED=True
 Enable Taiga [GitHub](https://github.com) importer. Remember to enable it in the front client too. Requires GitHub client ID and secret.
 
 Examples:
-```
+```yml
 TAIGA_IMPORTER_GITHUB_ENABLED=False
 ```
-```
+```yml
 TAIGA_IMPORTER_GITHUB_ENABLED=True
 TAIGA_IMPORTER_GITHUB_CLIENT_ID=XXXXXX_get_a_valid_client_id_from_github_XXXXXX
 TAIGA_IMPORTER_GITHUB_CLIENT_SECRET=XXXXXX_get_a_valid_client_secret_from_github_XXXXXX
@@ -361,10 +327,10 @@ TAIGA_IMPORTER_GITHUB_CLIENT_SECRET=XXXXXX_get_a_valid_client_secret_from_github
 Enable Taiga [Trello](https://trello.com/) importer. Remember to enable it in the front client too. Requires Trello API key and secret.
 
 Examples:
-```
+```yml
 TAIGA_IMPORTER_TRELLO_ENABLED=False
 ```
-```
+```yml
 TAIGA_IMPORTER_TRELLO_ENABLED=True
 TAIGA_IMPORTER_TRELLO_API_KEY=XXXXXX_get_a_valid_api_key_from_trello_XXXXXX
 TAIGA_IMPORTER_TRELLO_API_SECRET=XXXXXX_get_a_valid_secret_key_from_trello_XXXXXX
@@ -377,10 +343,10 @@ TAIGA_IMPORTER_TRELLO_API_SECRET=XXXXXX_get_a_valid_secret_key_from_trello_XXXXX
 Enable Taiga [JIRA](https://www.atlassian.com/software/jira) importer. Remember to enable it in the front client too. Requires JIRA consumer key and valid certificate.
 
 Examples:
-```
+```yml
 TAIGA_IMPORTER_JIRA_ENABLED=False
 ```
-```
+```yml
 TAIGA_IMPORTER_JIRA_ENABLED=True
 TAIGA_IMPORTER_JIRA_CONSUMER_KEY=XXXXXX_get_a_valid_consumer_key_from_jira_XXXXXX
 TAIGA_IMPORTER_JIRA_CERT=XXXXXX_get_a_valid_cert_from_jira_XXXXXX
@@ -394,10 +360,10 @@ TAIGA_IMPORTER_JIRA_PUB_CERT=XXXXXX_get_a_valid_pub_cert_from_jira_XXXXXX
 Enable Taiga [Asana](https://asana.com) importer. Remember to enable it in the front client too. Requires Asana App ID and secret.
 
 Examples:
-```
+```yml
 TAIGA_IMPORTER_ASANA_ENABLED=False
 ```
-```
+```yml
 TAIGA_IMPORTER_ASANA_ENABLED=True
 TAIGA_IMPORTER_ASANA_APP_ID=XXXXXX_get_a_valid_app_id_from_asana_XXXXXX
 TAIGA_IMPORTER_ASANA_APP_SECRET=XXXXXX_get_a_valid_app_secret_from_asana_XXXXXX
@@ -410,10 +376,10 @@ TAIGA_IMPORTER_ASANA_APP_SECRET=XXXXXX_get_a_valid_app_secret_from_asana_XXXXXX
 Enable [Taiga Events](https://github.com/Monogramm/docker-taiga-events). Requires RabbitMQ.
 
 Examples:
-```
+```yml
 TAIGA_EVENTS_ENABLED=False
 ```
-```
+```yml
 TAIGA_EVENTS_ENABLED=True
 RABBIT_USER=guest
 RABBIT_PASSWORD=guest
@@ -435,10 +401,10 @@ RABBIT_PORT=5672
 Enable Taiga asynchronous mode. Requires Redis, Celery and RabbitMQ.
 
 Examples:
-```
+```yml
 TAIGA_ASYNC_ENABLED=False
 ```
-```
+```yml
 TAIGA_ASYNC_ENABLED=True
 RABBIT_USER=guest
 RABBIT_PASSWORD=guest
@@ -447,7 +413,7 @@ RABBIT_PORT=5672
 REDIS_HOST=redis
 REDIS_PORT=6379
 ```
-```
+```yml
 TAIGA_ASYNC_ENABLED=True
 RABBIT_USER=taiga
 RABBIT_PASSWORD=somethingverysecure

--- a/docker-compose.yml
+++ b/docker-compose.yml
@@ -1,0 +1,173 @@
+version: '2'
+
+services:
+  taigadb:
+    image: postgres:10-alpine
+    container_name: taigadb
+    restart: always
+    ports:
+      - 5432:5432
+    volumes:
+      - /srv/taiga/db/data:/var/lib/postgresql/data
+    environment:
+      - POSTGRES_DB=taiga
+      - POSTGRES_USER=taiga
+      - POSTGRES_PASSWORD=taiga
+
+  taiga_back:
+    build: ./images/${VERSION}/${VARIANT}
+    #image: monogramm/docker-taiga-back-base:${VERSION}-${VARIANT}
+    #hostname: taiga.${DOMAIN}
+    container_name: taiga_back
+    restart: always
+    depends_on:
+      - taigadb
+    ports:
+      - 8001:8001
+    volumes:
+      # Media and uploads directory. Required (or you will lose all uploads)
+      - /srv/taiga/back/media:/usr/src/taiga-back/media
+      - /srv/taiga/back/static:/usr/src/taiga-back/static
+      # Taiga configuration directory. Makes it easier to change configuration with your own
+      #- /srv/taiga/back/conf:/taiga
+      - /etc/localtime:/etc/localtime:ro
+      - /etc/timezone:/etc/timezone:ro
+    environment:
+      # Your hostname (REQUIRED)
+      - TAIGA_HOSTNAME=localhost
+      #- TAIGA_HOSTNAME=taiga.${DOMAIN}
+      #- TAIGA_SSL=False
+      #- TAIGA_SSL_BY_REVERSE_PROXY=True
+      # Secret key for cryptographic signing
+      #- TAIGA_SECRET_KEY=${TAIGA_SECRET}
+      # Database settings
+      - TAIGA_DB_HOST=taigadb
+      - TAIGA_DB_NAME=taiga
+      - TAIGA_DB_USER=taiga
+      - TAIGA_DB_PASSWORD=taiga
+      # when the db comes up from docker, it is usually too quick
+      - TAIGA_SLEEP=2
+      # To use an external SMTP for emails, fill in these values:
+      - TAIGA_ENABLE_EMAIL=False
+      #- TAIGA_EMAIL_FROM=taiga@${TAIGA_EMAIL_DOMAIN}
+      #- TAIGA_EMAIL_USE_TLS=True
+      #- TAIGA_EMAIL_HOST=smtp.${TAIGA_EMAIL_DOMAIN}
+      #- TAIGA_EMAIL_PORT=587
+      #- TAIGA_EMAIL_USER=${TAIGA_SMTP_USER}
+      #- TAIGA_EMAIL_PASS=${TAIGA_SMTP_PWD}
+      # Backend settings
+      - TAIGA_DEBUG=False
+      - TAIGA_PUBLIC_REGISTER_ENABLED=False
+      - TAIGA_FEEDBACK_ENABLED=False
+      #- TAIGA_FEEDBACK_EMAIL=taiga@${TAIGA_EMAIL_DOMAIN}
+      # Events settings
+      - TAIGA_EVENTS_ENABLED=True
+      #- RABBIT_USER=${TAIGA_RABBIT_USER}
+      #- RABBIT_PASSWORD=${TAIGA_RABBIT_PASSWORD}
+      - RABBIT_HOST=taiga_rabbit
+      - RABBIT_PORT=5672
+      # Async settings
+      # To enable async mode, uncomment the following lines:
+      #- TAIGA_ASYNC_ENABLED=True
+      #- REDIS_HOST=taiga_redis
+      #- REDIS_PORT=6379
+
+  taiga_front:
+    #build: ./images/${VERSION}/${VARIANT}
+    image: monogramm/docker-taiga-front-base:${VERSION}-${VARIANT}
+    #hostname: taiga.${DOMAIN}
+    container_name: taiga_front
+    restart: always
+    depends_on:
+      - taiga_back
+      # To disable taiga-events, comment the following lines:
+      - taiga_events
+      - taiga_rabbit
+      # To enable async mode, uncomment the following lines:
+      #- taiga_redis
+    ports:
+      - 80:80
+      - 443:443
+    volumes:
+      # Media and uploads directory. Required for NGinx
+      - /srv/taiga/back/media:/usr/src/taiga-back/media:ro
+      - /srv/taiga/back/static:/usr/src/taiga-back/static:ro
+      # Taiga configuration directory. Makes it easier to change configuration with your own
+      #- /srv/taiga/front/conf:/taiga
+      - /etc/localtime:/etc/localtime:ro
+      - /etc/timezone:/etc/timezone:ro
+    environment:
+      # Your hostname (REQUIRED)
+      - TAIGA_HOSTNAME=localhost
+      #- TAIGA_HOSTNAME=taiga.${DOMAIN}
+      #- TAIGA_SSL=True
+      #- TAIGA_SSL_BY_REVERSE_PROXY=True
+      # Frontend settings
+      - TAIGA_DEBUG=false
+      - TAIGA_DEBUG_INFO=false
+      - TAIGA_DEFAULT_LANGUAGE=en
+      - TAIGA_DEFAULT_THEME=taiga
+      - TAIGA_PUBLIC_REGISTER_ENABLED=false
+      - TAIGA_FEEDBACK_ENABLED=True
+      - TAIGA_SUPPORT_URL=https://tree.taiga.io/support
+      - TAIGA_PRIVACY_POLICY_URL=
+      - TAIGA_TOS_URL=
+      - TAIGA_GDPR_URL=
+      - TAIGA_MAX_UPLOAD_SIZE=104857600
+      - TAIGA_CONTRIB_PLUGINS='slack cookie-warning'
+      #- TAIGA_CONTRIB_PLUGINS='slack cookie-warning gitlab-auth github-auth'
+      - TAIGA_GRAVATAR=true
+      - TAIGA_LOGIN_FORM_TYPE=ldap
+      # Backend settings
+      - TAIGA_BACK_HOST=taiga_back
+      - TAIGA_BACK_PORT=8001
+      # Events settings
+      - TAIGA_EVENTS_ENABLED=True
+      - TAIGA_EVENTS_HOST=taiga_events
+      - TAIGA_EVENTS_PORT=8888
+
+  # To disable taiga-events, comment all the following lines:
+  taiga_rabbit:
+    image: rabbitmq:3-alpine
+    hostname: taiga_rabbit
+    container_name: taiga_rabbit
+    ports:
+      - 5672:5672
+    #environment:
+    #  - RABBITMQ_DEFAULT_USER=${TAIGA_RABBIT_USER}
+    #  - RABBITMQ_DEFAULT_PASS=${TAIGA_RABBIT_PASSWORD}
+
+  taiga_events:
+    image: monogramm/docker-taiga-events:alpine
+    container_name: taiga_events
+    links:
+      - taiga_rabbit
+    ports:
+      - 8888:8888
+    environment:
+      #- RABBIT_USER=${TAIGA_RABBIT_USER}
+      #- RABBIT_PASSWORD=${TAIGA_RABBIT_PASSWORD}
+      - RABBIT_HOST=taiga_rabbit
+      - RABBIT_PORT=5672
+      - TAIGA_EVENTS_SECRET=${TAIGA_SECRET}
+      - TAIGA_EVENTS_PORT=8888
+
+  # To enable async mode, uncomment the following lines:
+#  taiga_redis:
+#    image: redis:4.0-alpine
+#    container_name: taiga_redis
+#    restart: always
+#    ports:
+#      - 6379:6379
+#    volumes:
+#      - /srv/taiga/redis/:/data
+#      - /etc/localtime:/etc/localtime:ro
+#      - /etc/timezone:/etc/timezone:ro
+#
+#  # FIXME Celery docker image is deprecated since 2017
+#  taiga_celery:
+#    image: celery
+#    container_name: taiga_celery
+#    links:
+#      - taiga_rabbit
+#      - taiga_redis

--- a/docker-compose.yml
+++ b/docker-compose.yml
@@ -46,7 +46,7 @@ services:
       - TAIGA_DB_USER=taiga
       - TAIGA_DB_PASSWORD=taiga
       # when the db comes up from docker, it is usually too quick
-      - TAIGA_SLEEP=2
+      - TAIGA_SLEEP=5
       # To use an external SMTP for emails, fill in these values:
       - TAIGA_ENABLE_EMAIL=False
       #- TAIGA_EMAIL_FROM=taiga@${TAIGA_EMAIL_DOMAIN}
@@ -149,7 +149,7 @@ services:
       #- RABBIT_PASSWORD=${TAIGA_RABBIT_PASSWORD}
       - RABBIT_HOST=taiga_rabbit
       - RABBIT_PORT=5672
-      - TAIGA_EVENTS_SECRET=${TAIGA_SECRET}
+      #- TAIGA_EVENTS_SECRET=${TAIGA_SECRET}
       - TAIGA_EVENTS_PORT=8888
 
   # To enable async mode, uncomment the following lines:

--- a/images/4.0/alpine/Dockerfile
+++ b/images/4.0/alpine/Dockerfile
@@ -1,4 +1,4 @@
-FROM python:3.7-alpine
+FROM python:3.6-alpine
 
 LABEL maintainer="Mathieu BRUNOT <mathieu.brunot at monogramm dot io>"
 

--- a/images/4.1/alpine/Dockerfile
+++ b/images/4.1/alpine/Dockerfile
@@ -1,4 +1,4 @@
-FROM python:3.7-alpine
+FROM python:3.6-alpine
 
 LABEL maintainer="Mathieu BRUNOT <mathieu.brunot at monogramm dot io>"
 

--- a/images/4.2/alpine/Dockerfile
+++ b/images/4.2/alpine/Dockerfile
@@ -1,4 +1,4 @@
-FROM python:3.7-alpine
+FROM python:3.6-alpine
 
 LABEL maintainer="Mathieu BRUNOT <mathieu.brunot at monogramm dot io>"
 

--- a/update.sh
+++ b/update.sh
@@ -7,7 +7,7 @@ declare -A base=(
 
 declare -A python_versions=(
 	[3]='3.6'
-	[4]='3.7'
+	[4]='3.6'
 )
 
 variants=(

